.github/workflows/pr-assign-team-label.yml

```diff
@@ -0,0 +1,16 @@
+name: Assign PR team labels
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
+        with:
+          repo-token: ${{ secrets.GH_ACCESS_TOKEN }}
+          organization-name: calcom
+          ignore-labels: "insights, i18n, organizations, workflows, embed, webhooks, event types, routing forms, billing, ci, docs, ui, booking page, api, automated tests, seats, bookings, caldav, teams, performance, emails, app store, authentication"
```

Comment:
hmmm ignoring wont work... i wonder if we can instead fork this action and limit it only to "cal-com" label?